examples/examples_py_test.go

```diff
@@ -44,9 +44,8 @@ func TestAccAlbLegacyPy(t *testing.T) {
 					RunUpdateTest: true,
 					EditDirs: []integration.EditDir{
 						{
-							Dir:             "step2",
-							Additive:        true,
-							ExpectNoChanges: true,
```

Comment:
Why does this change?

Reply:
Subnets are now a computed field so changes outside of the RouteTable will cause diffs